org.mixxx.Mixxx.yaml

```diff
@@ -3,38 +3,53 @@ runtime: org.kde.Platform
 runtime-version: '5.15-23.08'
 sdk: org.kde.Sdk
 command: mixxx
+rename-icon: mixxx
+
+# Tag and mark as beta until 2.4 final release
+tags:
+  - beta
+desktop-file-name-suffix: ' (beta)'
+
 finish-args:
-  - --share=ipc
+  # Required for DJ controller support
+  - --device=all
+
+  # Wayland is used by default
   - --socket=wayland
-  - --socket=x11
-  # JACK support via PipeWire
-  - --filesystem=xdg-run/pipewire-0
-  # PulseAudio isn't actually helpful for Mixxx, but this allows access to ALSA
-  # devices too since https://github.com/flatpak/flatpak/pull/3663
-  - --socket=pulseaudio
-  # Only used for MusicBrainz lookups
+  - --socket=fallback-x11
```

Comment:
do you test the plugins?

Reply:
I mean their UI. On a Wayland session.

Reply:
The question is rhetorical. It's just to show you how this breaks.

Reply:
Including the plugins UI?

Reply:
Mixxx does not support LV2 plugin GUIs, so the point is moot.